source/dotnet/Library/AdaptiveCards/AdaptiveInlinesConverter.cs

```diff
@@ -24,6 +24,11 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
             var array = JArray.Load(reader);
             List<object> list = array.ToObject<List<object>>();
             List<AdaptiveInline> arrayList = new List<AdaptiveInline>();
+            var serializerSettigns = new JsonSerializerSettings
+            {
+                ContractResolver = new WarningLoggingContractResolver(new AdaptiveCardParseResult(), ParseContext),
+                Converters = { new StrictIntConverter() }
```

Comment:
>StrictIntConverter [](start = 35, length = 18)

Question: Why this?

Reply:
I don't have particular reason for using the converter. I'm following the usage shown in AdaptiveCard.cs LN239-242. I couldn't find reasons to differ.

---
In reply to: [341414800](https://github.com/microsoft/AdaptiveCards/pull/3556#discussion_r341414800) [](ancestors = 341414800)